### PR TITLE
OEA-449: date of birth with day lower than 12 changes the date of birth automatically with Other and Referral as source

### PR DESCRIPTION
**Pull request by marshl**
I'm not sure that this is the best solution. There is an increased risk of side-effects by including other save and validate code. Conversely, there would be an increased risk of a _lack_ of side effects if the date format was massaged manually.

### CODE REVIEW

**Comment by marshl** [comment]
PatientAgeParameterTest::testSearchSingleInput() has failed again. Will rebase and merge on fixed branch once advised. Please stand by.

**Comment by apigram** [comment]
@Vegemash Personally I think we should be testing against patients whose date of birth is NOT the 1st of January as well. Diverse fixtures mean more confidence in the tests.

**Comment by marshl** [comment]
The branch has been rebuilt to not include commits for OEA-197

**Review by apigram — APPROVED** [review]

File: protected/tests/fixtures/patient.php [hunk 1]
```diff
@@ -23,7 +23,7 @@
         'title' => 'Mr.',
         'first_name' => 'John',
         'last_name' => 'Jones',
-        'dob' => (new DateTime())->modify('-47 year')->format('Y-m-d'),
+        'dob' => (new DateTime())->modify('-47 year')->format('Y').'-01-01',
```
Comment (marshl) [inline protected/tests/fixtures/patient.php]: @Vegemash Alternatively you could use ->format('Y-01-01')

**Pull request merged**